README.md

```diff
@@ -168,6 +168,51 @@ python3 -m pip install --user -r requirements.txt
 
 Each example in samples folder demonstrates different ways in which OMRChecker can be used.
 
+
+_**Issues you might face while installing :**_
+<details>
+<summary>Issues</summary>
+<br>
+#1.<ERROR: Could not open requirements file: [Errno 2] No such file or directory: 'requirements.txt'> On Windows    after using the command: 
+	
+	
+	$ python3 -m pip install --user -r requirements.txt
+	
+	
+	
+![P1](https://user-images.githubusercontent.com/111011580/193443625-04798d34-98ba-4d28-b3db-0975be936289.png)
+	
+	
+ **Solution-->**
+	
+You can resolve it by following the steps : 
+
+--> It can happen if you are not in the directory where you have cloned your project .
+	
+
+ 1. Navigate to the directory where you have cloned your project .
+ 
+Example -  I had made OMR Checker Directory to clone the project ,to navigate to the directory use cd <Name of Directory>
+ 
+![cd](https://user-images.githubusercontent.com/111011580/193441708-b133dd86-c85b-4da0-b140-3c50a81abac8.png)
```

Comment:
Let's avoid screenshots here. We can link to the github issues for details, we should only briefly mention the issue here. And redirect to solution there.